README.md

```diff
@@ -22,137 +22,49 @@ Thimble requires a modern web browser, and we recommend using Mozilla Firefox or
-
-## Getting Ready to Publish
-To publish locally, you'll need to do the following...
-
-**1. Teach the ID server about the Publish server**
-
-* Run ``createdb webmaker_oauth_test`` to create a test database
-* In your id.webmaker.org folder
-  * Run ``node scripts/create-tables.js``
-  * Edit ``scripts/test-data.sql`` and replace it's contents with:
-
-      ```sql
-        INSERT INTO clients VALUES
-          ( 'test',
-            'test',
-            '["password", "authorization_code"]'::jsonb,
-            '["code", "token"]'::jsonb,
-            'http://localhost:3500/callback' )
-      ```
-
-  * Run ``node scripts/test-data.js``
-    * You'll see a ``INSERT 0 1`` message if successful
-
-**2. Set up the local data folder**
-
-Instead of publishing to Amazon AWS, we'll be publishing to a local folder. Perform the following steps to set this up.
-* Run ``npm install -g http-server && mkdir -p /tmp/mox/test && cd /tmp/mox/test && http-server -p 8001``
-* Run ``cd /tmp/mox/test && http-server -p 8001`` to start the server
-* In your publish.webmaker.org folder
-  * Open the ``.env`` file
-  * Make sure that ``PUBLIC_PROJECT_ENDPOINT="localhost:8001"``` is set as shown here
-  * Restart publish server
-
-**3. Sign In**
-
-To publish locally, you'll need an account.
-* Go to [http://localhost:3000/account](http://localhost:3000/account)
-* Click ``Join Webmaker`` and complete the process, you can use a fake email
-* When you've created your account, click ``Set permanent password instead``
-  * This lets you authenticate your account without needing email
-* Go back to Thimble and Log In with your new account
-
-## Running the parts
-This is the list of commands to get each part up and running.
-
-* Thimble ``npm start``
-* Bramble ``npm start``
-* PostgreSQL Database ``postgres -D /usr/local/var/postgres``
-* Webmaker ID server ``npm start``
-* Webmaker Login Server ``npm start``
-* Webmaker Publishing Server ``npm start``
-* Local publish folder ``cd /tmp/mox/test && http-server -p 8001``
-
-Development additionals
------------------------
-
-We handle JS, HTML and CSS linting through grunt, which is very simple
-to set up if you don't have it installed already:
-
-```npm install -g grunt-cli```
-
-After this, simpy run ```grunt``` before commiting code and you should
-be good to go.
-
-Building the front-end
-----------------------
+### Requirements
+
+- Node.js (version 4.6 or later) [[download](https://nodejs.org/en/download/)]
+- Vagrant (version 1.9 or later) [[download](https://www.vagrantup.com/downloads.html)]
+- Virtualbox (version 5.1 or later) [[download](https://www.virtualbox.org/wiki/Downloads)]
+
+The setup of Thimble can be divided into two distinct sections:
+
+### Editor
+
+To fully use Thimble locally, you need to first setup Brackets locally first. This can be done by following the steps outlined below:
+
+- Fork and clone the [Brackets repository](https://github.com/mozilla/brackets)
```

Comment:
Following these instructions to install brackets causes ```npm run build``` to fail because the git sub-modules haven't been installed. Doing ```git clone https://github.com/mozilla/brackets --recursive``` clones the repository and installs the subsequent git sub-modules permitting a successful NPM build.